src/main/features/claim/form/models/eligibility/over18Defendant.ts

```diff
@@ -0,0 +1,27 @@
+import { YesNoOption } from 'models/yesNoOption'
+import { PartyType } from 'app/common/partyType'
+
+export class Over18Defendant {
```

Comment:
Please rename this as `Over18Defendant` does not describe the purpose of it well. I would recommend `DefendantAgeOption` or `DefendantAgeEligibilityOption`